app/src/main/res/values/strings.xml

```diff
@@ -49,4 +49,14 @@
     <string name="see_full_bio">See full bio >></string>
     <string name="see_less">See less >></string>
 
+    <!-- Search -->
+    <string name="search">Search</string>
+    <string name="movies_or_series">Movies or Series</string>
+    <string name="movie">Movie</string>
+    <string name="job_actor">Actor</string>
+    <string name="job_director">Director</string>
+    <string name="job_writer">Writer</string>
+    <string name="job_crew">Crew</string>
+    <string name="empty_result_multi_search">your search didn’t have any results</string>
```

Comment:
naming convention.
please check [this](https://github.com/adessoTurkey/android-guideline/blob/master/naming_convention.md#strings) out